backend/development.ini.sample

```diff
@@ -490,21 +515,27 @@ caldav.radicale.headers.Access-Control-Expose-Headers = Etag
 ; search.elasticsearch.port = 9200
 # global elasticsearch timeout in seconds
 ; search.elasticsearch.request_timeout = 60
+
 # main index alias use to search/index content in elasticsearch
-; search.elasticsearch.index_alias = tracim_contents
+# search.elasticsearch.index_alias = tracim_contents
```

Comment:
Same for parameter here:
- `; search.elasticsearch.index_alias = `

For this paramter I'm not sure if I understand correctly. No default value exist?

Reply:
fixed